src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/BuiltinOperatorMethodSymbol.cs

```diff
@@ -0,0 +1,227 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
+{
+    // Describes a conversion operator method from containingType to returnType as if it were defined like so:
+    // public class/struct ContainingType
+    // {
+    //     public static explicit operator ReturnType(ContainingType value) => ..;
+    // }
+    internal class BuiltinOperatorMethodSymbol : IMethodSymbol
+    {
+        public BuiltinOperatorMethodSymbol(ITypeSymbol returnType, INamedTypeSymbol containingType)
+        {
+            ReturnType = returnType;
+            Parameters = new IParameterSymbol[]
+            {
+                new BuiltinOperatorParameterSymbol(containingType),
+            }.ToImmutableArray();
+            ContainingType = containingType;
+        }
+
+        public ITypeSymbol ReturnType { get; }
+
+        public INamedTypeSymbol ContainingType { get; }
+
+        public ImmutableArray<IParameterSymbol> Parameters { get; }
+
+        public string? GetDocumentationCommentXml(CultureInfo? preferredCulture = null, bool expandIncludes = false, CancellationToken cancellationToken = default)
+        {
+            // Explicit conversion of <see cref="T:{0}"/> to <see cref="T:{1}"/>.
+            var template = @$"
+<summary>
+    {CSharpFeaturesResources.Explicit_conversion_of_see_cref_T_0_to_see_cref_T_1}
+</summary>
+";
+            // "global::" is added by the cref formatter.
```

Comment:
not sure what this comment means.

Reply:
It's explaining `SymbolDisplayGlobalNamespaceStyle.Omitted`. Otherwise further down in the description service types are formatted like `global::global::System.Object`